documentation/MeetingMinutes/2024/MOM-2024-01-31.md

```diff
@@ -0,0 +1,169 @@
+- Axel asks for clarification on whether scope is not needed or purpose is not needed. Clarified that scope is always required. Axel asks if charging models can be differentiated by different scopes and Elizabeth points out that it's not possible. Candidate to be discussed in v0.3.
+
+<br>
+
+Regarding 1) "Format of the profile":
+- Axel points out that the main argument against a self-contained format is that it's very difficult to see what changes the standard. Points out that if we have a full document, we should clearly identify each change that changes/updates/deviates the standard.
+- Jesús points out that Telefónica first has tried to create a profile like Mobile Connect (which is self-contained), as requested in recent calls. On the other hand, Telefónica thinks that if we add references to standards and then the standard is not completely clear on a topic, different topics may have different interpretations. We have had this problem in the past. In any case, we agree that it does not make sense to copy everything from another reused standard, but only the relevant parts.
+- Axel will be fine with a note at the beginning explaining why we do it one way or the other. For example, explain that "if we repeat redirect_url and it's not clearly marked as different, a reader will have to go to auth2 to check if the text is the same or has changed". Indicates that if we choose to repeat, we should clearly indicate what has changed. Indicates that FAPI goes a different way, just with certain restrictions.
+- Shilpa indicates that she has no other opinion. Probably pros and cons of each option. Elizabeth would vote for a self-contained with a list of changes (change history), otherwise it wouldn't be possible to follow. Axel points out that if you're a developer reading this, you won't notice what's changed from the existing standard. Elizabeth points out that with a delta it's going to be impossible, she has 20 years experience in development.... Axel has been working in standards for 10 years, and mistakes happen, he thinks that if you rewrite something, you might invalidate the other standard.
+- Toyeeb (GSMA) indicates that a self-contained document will be more appreciated by dev community, based on current experience with developers so far in GSMA.
+- Axel points out that it's fine as long as it's clear what changes are included and why. He warns that we need to be really careful. Fabio points out that maybe we can point out the text that changes from the standard, he agrees with Axel in this case
+- **After asking to the WG participants considering the different opinions, the way forward agreed is:  go with self-contained + compromise to properly note any change in relation to the reused standard.** 
```

Comment:
Shilpa's suggestion to collect organization specific viewpoint was not taken into consideration.

Reply:
I strongly disagree, sorry. To say that you were ignored, I don't think that's true, and I can't put that in the minutes.

As stated in https://github.com/camaraproject/IdentityAndConsentManagement/pull/113#issuecomment-1921778653, there have been days of feedback on the profile format since the PR was created before this meeting call, there have been different opinions, and the meeting last week was precisely to agree on a way forward and unblock this hot topic 

During the call, as noted in the meeting minutes, other participants such as Ericsson, GSMA and Telefónica also gave their opinions. And in this call with other participants from other connected companies, it was also requested whether we all agree to move forward with a self-contained approach. And there was an explicit agreement in that call to do so. Not all participants did provide an anwser, of course, but lazy agreements apply to CAMARA.

Reply:
How do we assume a lazy consensus if we don't give some time to at least the people who are active in the PR review or better still the participants of the WG  to give their feedback/opinion? This was done on the spot during the meeting with just the participants who attended the meeting that day and further limiting it to the few who spoke. It would be very helpful if we collect the feedbacks before we at least make major decisions.

Reply:
@shilpa-padgaonkar PR was created 25/01 and WG meeting was 31/01. That is 4-5 working days in between. The PR was quite active and many people gave feedback to the PR and specifically this topic was identified as one of the two main discussion tracks and therefore it was suggested to be discussed in the meeting call. Especially because it had a big impact on the PR because the PR content would be totally different depending on the format choice (delta or self-contained). So we had to agree on the profile format in order to proceed with the PR content and the rest of the comments and discussions.
I don't like what you are suggesting with your comments and it is clear that we have different opinions. I would have liked to hear your comment above in the meeting itself and not now. But as I said before, if the WG wants to revisit the format decision, then let's do it. But this is not a unilateral decision by Telefonica or anything like that, I want to make that very clear. So please, let's ask the rest of the participants to revisit the decision and let's set the deadline that you think meets your expectations and the expectations of the WG.